content_main.js

```diff
@@ -9,6 +27,27 @@ main();
 async function main() {
 	const siteIsMonetized = await isMonetizationPresent();
 	console.log("siteIsMonetized: ", siteIsMonetized);
+
+	const paymentPointer = maybeGetPaymentPointer();
```

Comment:
`maybeGetPaymentPointer()` is already being called in `isMonetizationPresent()` - we should only call it once and store the payment pointer

Reply:
The payment pointer may be changed by the page, so it is good to retrieve the payment pointer every time we use it. That being said, we will need to add some code so that the payment pointer is validated when and only when it changes and has not already been validated.

Reply:
Let's set a timeout for how long we consider a payment pointer to be "valid". We should still regularly validate pointers even if we've already validated them because we may not be able to ensure the validity of the endpoint after a certain amount of time.